api/src/Common/Models/Race.cs

```diff
@@ -13,6 +13,8 @@ public class Race : IModel
         [Required]
         public DateTime Date { get; set; }
 
+        public string EventId { get; set; }
```

Comment:
This should probably be required, right?

Reply:
I could make it required but I'm not sure if that's always the case given the way we were mapping events to races. Specifically, when a user passes in a new race that didn't exist previously.

Reply:
actually nvm